pkg/pgmodel/ingestor/trace/writer.go

```diff
@@ -59,7 +61,10 @@ type traceWriterImpl struct {
 	tagCache     *clockcache.Cache
 }
 
-func NewWriter(conn pgxconn.PgxConn) *traceWriterImpl {
+func NewWriter(conn pgxconn.PgxConn, t telemetry.Engine) *traceWriterImpl {
+	if err := t.RegisterMetric("promscale_ingested_spans_total", metrics.IngestorItems.With(prometheus.Labels{"type": "trace", "kind": "span"})); err != nil {
```

Comment:
This metric is not aligned with the recent consistent metrics design doc. Please adjust.